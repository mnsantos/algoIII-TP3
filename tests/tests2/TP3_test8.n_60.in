60 561
3 10
1 8
1 2
6 10
1 7
7 10
1 3
7 9
2 7
8 10
6 7
1 4
6 14
1 16
1 17
1 18
1 9
7 14
3 9
1 6
1 12
1 5
2 16
1 10
3 15
6 8
5 13
4 9
1 11
2 15
3 6
57 58
56 59
56 58
55 59
54 59
54 55
53 59
52 56
51 60
51 59
51 58
51 56
51 55
51 52
50 58
50 55
49 60
49 56
49 53
49 52
48 59
48 58
48 55
48 53
48 52
47 57
47 54
47 52
47 51
46 59
46 58
46 53
46 50
46 49
46 48
45 55
45 48
44 58
44 48
43 58
43 54
43 48
43 46
43 45
42 60
42 57
42 52
42 51
42 50
42 47
42 46
42 45
42 43
41 60
41 52
41 51
41 45
41 44
41 43
40 59
40 51
40 50
40 49
39 57
39 51
39 49
39 48
39 47
38 59
38 56
38 52
38 50
38 47
38 46
38 44
38 41
38 40
37 60
37 54
37 52
37 51
37 50
37 47
37 44
37 42
37 38
36 49
36 46
36 44
36 39
36 38
36 37
35 59
35 48
35 42
35 36
34 58
34 56
34 52
34 49
34 47
34 44
34 42
34 39
34 35
33 59
33 50
33 47
33 46
33 41
33 34
32 59
32 58
32 57
32 56
32 54
32 53
32 48
32 45
32 36
31 60
31 55
31 54
31 42
31 39
31 34
30 60
30 58
30 55
30 54
30 50
30 45
30 41
30 40
30 38
30 37
30 34
29 60
29 58
29 57
29 56
29 55
29 52
29 51
29 49
29 48
29 46
29 45
29 42
29 40
29 39
29 38
28 57
28 56
28 54
28 48
28 43
28 30
27 59
27 53
27 51
27 48
27 47
27 45
27 43
27 42
27 40
27 34
27 33
27 28
26 60
26 59
26 57
26 54
26 51
26 50
10 22
10 18
9 60
9 59
9 56
9 55
9 54
9 50
9 48
9 41
9 33
9 30
9 28
9 23
9 20
9 18
9 17
9 15
8 60
8 53
8 52
8 51
8 50
8 49
8 46
8 45
8 44
8 42
8 40
8 37
8 29
8 22
8 21
8 20
8 16
8 15
7 56
7 53
7 51
7 50
7 48
7 46
7 42
7 39
7 37
7 32
7 30
7 27
7 19
7 18
6 51
6 45
6 43
6 39
6 38
6 25
6 22
6 19
6 16
5 60
5 52
5 51
5 49
5 47
5 43
5 42
5 39
5 36
5 34
5 32
5 28
5 27
5 24
5 19
5 18
5 17
4 54
4 53
4 50
4 47
4 44
4 42
4 39
4 35
4 32
4 28
4 26
1 50
1 49
1 48
1 47
1 46
1 45
1 44
1 43
1 42
1 41
1 40
1 39
1 38
1 37
1 36
1 35
1 34
1 33
1 32
1 31
1 30
1 29
1 28
1 27
1 19
1 20
1 21
1 22
1 23
1 24
1 25
1 26
1 51
1 52
1 53
1 54
1 55
1 56
1 57
1 58
1 59
1 60
2 26
2 28
2 32
2 36
2 42
2 43
2 50
2 60
3 22
3 23
3 24
3 32
3 34
3 35
3 36
3 38
3 40
3 41
3 42
3 47
3 48
3 49
3 51
3 58
3 59
4 18
10 30
10 31
10 32
10 34
10 42
10 44
10 46
10 47
10 55
10 59
11 13
11 17
11 19
11 20
11 27
11 28
11 29
11 33
11 39
11 40
11 41
11 42
11 46
11 53
11 57
11 59
12 21
12 28
12 36
12 38
12 44
12 59
13 20
13 21
13 22
13 23
13 33
13 41
13 48
13 50
13 52
13 57
14 15
14 19
14 22
14 24
14 30
14 31
14 35
14 38
14 46
14 47
14 50
14 57
14 59
14 60
15 17
15 18
15 19
15 21
15 24
15 37
15 45
15 46
15 55
15 56
15 57
15 58
16 17
16 21
16 22
16 23
16 25
16 28
16 31
16 34
16 38
16 39
16 43
16 48
16 51
16 52
16 55
16 58
16 60
17 20
17 22
17 24
17 30
17 31
17 35
17 47
17 49
17 50
17 51
17 53
17 54
17 60
18 26
18 29
18 34
18 36
18 37
18 38
18 42
18 43
18 49
18 50
18 53
18 56
18 58
18 60
19 21
19 22
19 24
19 32
19 36
19 39
19 41
19 50
19 53
19 56
20 22
20 23
20 28
20 32
20 35
20 40
20 46
20 52
20 57
20 59
21 25
21 26
21 31
21 32
21 36
21 38
21 39
21 40
21 41
21 43
21 48
21 52
21 53
21 58
22 24
22 28
22 30
22 31
22 32
22 34
22 35
22 37
22 41
22 45
22 46
22 49
22 51
22 54
23 33
23 43
23 46
23 47
23 51
23 52
23 53
24 25
24 27
24 30
24 32
24 33
24 34
24 35
24 41
24 42
24 43
24 52
24 54
24 58
25 34
25 39
25 41
25 46
25 58
26 29
26 32
26 33
26 37
26 39
26 44
26 45
1 15
3 14
1 13
2 6
2 12
4 5
1 14
0
