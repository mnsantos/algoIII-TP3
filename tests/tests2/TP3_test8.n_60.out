141 6 1 8 29 42 51 52
